README.md

```diff
@@ -1654,6 +1654,9 @@ Sensors, 2024.
 *BeNeRF: Neural Radiance Fields from a Single Blurry Image and Event Stream*.
 - [Su et al., RAL 2025](#Su25ral),  
 *Motion-Aware Optical Camera Communication with Event Cameras*.
+- <a name="nkrumah2024ec"></a>Nkrumah, I., and Moshrefizadeh, M.,  Tahri, O., Blasch, E., Palaniappan, K., AliAkbarpour, H., 
```

Comment:
the key should be: nkrumah24sensors

Reply:
And please check the preview:
![image](https://github.com/user-attachments/assets/f7ea78ec-6ac1-4f1c-ad61-63a4325073b7)
a line break is missing.